tests/e2e/testcases/upgrade/suite.go

```diff
@@ -85,7 +86,64 @@ var _ = ginkgo.Describe("[Upgrade expect network failure]", ginkgo.Ordered, func
 	})
 })
 
-var _ = ginkgo.Describe("[Upgrade]", ginkgo.Ordered, func() {
+// upgrade a public network
+// the approach is rather simple: import the upgrade file,
+// call the apply command which "just" installs the file at an expected path,
+// and then check the file is there and has the correct content.
+var _ = ginkgo.Describe("[Upgrade public network]", ginkgo.Ordered, func() {
+	ginkgo.AfterEach(func() {
+		commands.CleanNetworkHard()
+		err := utils.DeleteConfigs(subnetName)
+		gomega.Expect(err).Should(gomega.BeNil())
+	})
+
+	ginkgo.It("can create and apply to public node", func() {
```

Comment:
do we need to be more clear regarding the upgrade that is beign performed? vm versus vs network upgrade...

Reply:
I don't think so, if you look further up the chain, this is part of a `ginkgo.Describe` which sets the context.